app/models/user.rb

```diff
@@ -11,8 +11,13 @@ class User < ActiveRecord::Base
   enum sam_status: { duns_blank: 0, sam_accepted: 1, sam_rejected: 2, sam_pending: 3 }
 
   def self.from_saml_omniauth(auth)
```

Comment:
this method should potentially be renamed or moved into its own service class. thoughts?